src/components/Footer/Footer.module.scss

```
@@ -1,7 +1,7 @@
+@use 'src/app/_variables';
```

これってちゃんと使えてるの？

基本こっちのはず

```suggestion
@use 'src/app/variables';
```